advanced/learning_to_rank.rst

```diff
@@ -0,0 +1,142 @@
+
+.. Copyright (C) 2011 Parth Gupta
+.. Copyright (C) 2016 Ayush Tomar
+
+
+=======================
+Xapian Learning-to-Rank
+=======================
+
+.. contents:: Table of Contents
+
+
+Introduction
+============
+
+Learning-to-Rank(LTR) can be viewed as a weighting scheme which involves machine learning. The main idea behind LTR is to bring up relevant documents given a low ranking by probablistic techniques like BM25 by using machine learning models. A model is trained by learning from the relevance judgements provided by a user corresponding to a set of queries and a corpus of documents. This model is then used to re-rank the matchset to bring more relevant documents higher in the ranking. Learning-to-Rank has gained immense popularity and attention among researchers recently. Xapian is the first project with Learning-to-Rank functionality added to it.
```

Comment:
> Xapian is the first project with Learning-to-Rank functionality added to it.

This seems quite a bold claim - for example, Terrier added support in 4.0 which looks like it was released in 2014.  Given we've not made a stable release of xapian-letor yet, I feel at best this might be technically correct, and I suspect in reality it's just not true.